js/sticky-bar.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+var $ = require('jquery');
+
+function StickyBar(selector) {
+  this.$body = $('body');
+  this.$bar = $(selector);
+  this.height = this.$bar.outerHeight();
+  this.offset = this.$bar.offset().top;
+  this.delay = 100; // Delay before it sticks
```

Comment:
`delay` makes me think time, but here you're actually using it as a distance offset. Unfortunately `offset` is already taken, but I would consider renaming this. Maybe `triggerOffset`?